packages/nx/src/adapter/ngcli-adapter.ts

```diff
@@ -1222,7 +1222,6 @@ function saveWorkspaceConfigurationInWrappedSchematic(
   );
   nxJson.generators = workspace.generators || workspace.schematics;
   nxJson.cli = workspace.cli || nxJson.cli;
-  nxJson.defaultProject = workspace.defaultProject;
```

Comment:
I'm not sure if this should be removed. If someone writes this into `angular.json`, we should move it over to `nx.json`